controlled_resource_handler.go

```diff
@@ -96,11 +97,10 @@ func (g *ControlledResourceHandler) OnDelete(obj interface{}) {
 // This method may be called with an empty controllerName.
 func (g *ControlledResourceHandler) rebuildControllerByName(logger *zap.Logger, namespace, controllerName, addUpdateDelete string) {
 	if controllerName == "" {
+		logger.Info("Object has no controller, so nothing was enqueued")
 		return
 	}
-	logger.
-		With(logz.ControllerName(controllerName)).
```

Comment:
Turns out, "Controller" is a really bad overused word.

I've found that:

- Controller means "kubernetes controller"
- Controller also means owner
- Controller also is a component in the SharedIndexInformer
- Controller is also the thing in MVC that we do in our APIServers.

:kappa: